api/models.py

```diff
@@ -127,6 +127,10 @@ class User(Base):
     logged_activities = db.relationship('LoggedActivity',
                                         backref='user',
                                         lazy='dynamic')
+
+    created_activities = db.relationship('Activity',
+                                         backref='user',
```

Comment:
I had missed this. Couldn't the backref be `added_by` instead?

Reply:
Done. changed the backref to `added_by`